src/main/java/nextstep/ladder/Ladder.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.ladder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Ladder {
+    private List ladder = new ArrayList<>();
+
+    public Ladder() {
+    }
+
+    public Ladder(int countOfPerson, int height) {
+        this.ladder = createLadder(countOfPerson, height);
```

Comment:
자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
        this(createLadder(countOfPerson, height));
```